ssm/emissions.py

```diff
@@ -53,6 +53,14 @@ def smooth(self, expected_states, variational_mean, data, input=None, mask=None,
         """
         raise NotImplementedError
 
+    def hessian_log_emissions_prob(self, data, input, mask, tag, x):
```

Comment:
You have to add `from autograd import hessian` at the top.

Reply:
Done.